src/Router.ts

```diff
@@ -924,6 +925,47 @@ export class Router extends EnhancedEventEmitter
 		}
 	}
 
+	/**
+	 * Create an ActiveSpeakerObserver
+	 */
+	async createActiveSpeakerObserver(
+		{
+			interval = 300,
+			appData = {}
+		}: ActiveSpeakerObserverOptions = {}
+	): Promise<ActiveSpeakerObserver>
+	{
+		logger.debug('createActiveSpeakerObserver()');
+		if (appData && typeof appData !== 'object')
+			throw new TypeError('if given, appData must be an object');
```

Comment:
Please let's keep coding conventions everywhere.

```suggestion
		logger.debug('createActiveSpeakerObserver()');

		if (appData && typeof appData !== 'object')
			throw new TypeError('if given, appData must be an object');
```